_pytest/config.py

```diff
@@ -364,6 +364,9 @@ def _importconftest(self, conftestpath):
                 _ensure_removed_sysmodule(conftestpath.purebasename)
             try:
                 mod = conftestpath.pyimport()
+                if hasattr(mod, 'pytest_plugins') and len(self._conftest_plugins) > 0:
+                    from _pytest.deprecated import PYTEST_PLUGINS_FROM_NON_TOP_LEVEL_CONFTEST
+                    self._warn(PYTEST_PLUGINS_FROM_NON_TOP_LEVEL_CONFTEST)
```

Comment:
we should include the relative module location in the error message

i'd appreciate if `warnings.warn` was used there, but i can understand leaving as is if it seems hard/very time consuming to enable it, i dont what to turn this into a time sink for you

Reply:
not a problem at all, I was just worried about its visibility

Reply:
done